packages/commonwealth/server/routes/getNewProfile.ts

```diff
@@ -61,7 +61,7 @@ const getNewProfile = async (
   });
 
   const addressIds = [...new Set<number>(addresses.map((a) => a.id))];
-  const threads = await models.Thread.findAll({
+  const threads = await models.Thread.scope('excludeAttributes').findAll({
```

Comment:
Changes to getNewProfile will likely break Profiles pages. Did not QA though.

Reply:
Again not breaking if it should on Frick

Reply:
Definitely broken
![2023-06-05-155633_1003x1018_scrot](https://github.com/hicommonwealth/commonwealth/assets/1860629/f7e0d3e9-f270-41b4-abb0-0c070eaff500)

Reply:
I see undefined! Actually I saw that too

Reply:
For read only screens, we can optimize too
we have the
- text - some query string format
- Plaintext - readable good for external api
- Body - markdown
- Version history

we should mention in acceptance criteria for new ticket to pull required columns only

Reply:
isThread ? body : comment.text

https://github.com/hicommonwealth/commonwealth/blob/ecac186ed8bdf2e75f2c01586c639d7a1338da67/packages/commonwealth/client/scripts/views/components/Profile/ProfileActivityRow.tsx#L124